Sources/tart/OCI/Registry.swift

```diff
@@ -83,7 +90,7 @@ struct TokenResponse: Decodable, Authentication {
   }
 
   func header() -> (String, String) {
-    ("Authorization", "Bearer \(token)")
+      return ("Authorization", "Bearer \(token ?? accessToken ?? "")")
```

Comment:
Removed extraneous indentation:

```suggestion
    return ("Authorization", "Bearer \(token ?? accessToken ?? "")")
```